src/functionaltest/java/com/ericsson/ei/frontend/TestSwitchBackend.java

```diff
@@ -0,0 +1,122 @@
+package com.ericsson.ei.frontend;
+
+import org.junit.*;
+import org.mockserver.client.server.MockServerClient;
+import org.mockserver.integration.ClientAndServer;
+
+import static org.mockserver.integration.ClientAndServer.startClientAndServer;
+
+import static org.junit.Assert.*;
+
+import static org.mockserver.model.HttpRequest.request;
+import static org.mockserver.model.HttpResponse.response;
+
+import java.io.File;
+import java.io.IOException;
+
+import com.ericsson.ei.frontend.pageobjects.AddBackendPage;
+import com.ericsson.ei.frontend.pageobjects.IndexPage;
+import com.ericsson.ei.frontend.pageobjects.InfoPage;
+import com.ericsson.ei.frontend.pageobjects.SubscriptionPage;
+import com.ericsson.ei.frontend.pageobjects.SwitchBackendPage;
+
+public class TestSwitchBackend extends SeleniumBaseClass{
+    private MockServerClient mockClient1;
+    private MockServerClient mockClient2;
+    private ClientAndServer mockServer1;
+    private ClientAndServer mockServer2;
+
+
+    private static final String BASE_URL = "localhost";
+    private static final int portServer1 = 8090;
```

Comment:
int constants values like a String constants also should be in uppercase

Reply:
Oh i agree :)